tests/tests_Sanity.cpp

```diff
@@ -19,7 +23,7 @@ TEST_SUITE("metall-ffi") {
             }
 
             {
-                auto *ptr = static_cast<size_t *>(metall_malloc(manager, obj_name, sizeof(size_t)));
```

Comment:
why do you alloc twice the size here?

Reply:
yes!

Reply:
If metall fails here to allocate a word we would also need to be always be very careful using it. 

This is blocker for updating metall.

Reply:
As I said, this has always been the case. Its not new

Reply:
sorry, didn't read carefully. well then, we can merge, I guess.

Reply:
As discussed I've reverted the manager settings to metall default, as they do not trigger this assertion